steam/docker-compose.yml

```
@@ -0,0 +1,59 @@
+services:
```

compose files should start with their version of the docker-compose specification, f.e.

```suggestion
version: "3.9"
services:
```